docs/user_guide/1_concepts.md

```diff
@@ -60,19 +60,13 @@ results = predictor.predict(
 
 For more information, see `landingai.common.InferenceMetadata` and `landingai.predict.Predictor.predict`.
 
-#### Cloud Inference Limitations
+#### Cloud Inference Rate Limit
 
-The `Predictor` class uses Cloud Deployment, which comes with below limitations.
+The `Predictor` class uses Cloud Deployment, which limits how often inference can be run. 
 
-1. **Rate Limit**
+You can run inference up to 40 times per minute per endpoint. If you exceed that limit, the API returns a `429 Too Many Requests` response status code. We recommend implementing an error handling or retry function in your application. If you have questions about inference limits, or need to run inference more frequently, please contact your Landing AI representative or [sales@landing.ai](mailto:sales@landing.ai).
```

Comment:
```suggestion
You can run inference up to 40 times per minute per endpoint. If you exceed that limit, the API returns a `429 Too Many Requests` response status code. If you have questions about inference limits, or need to run inference more frequently, please contact your Landing AI representative or [sales@landing.ai](mailto:sales@landing.ai).
```

I removed "We recommend implementing an error handling or retry function in your application." since the SDK actually implemented a retry function so they don't need to implement it. (Their retry is redundant if they implement one.)